force-app/main/default/classes/LVL_LevelEdit_CTRL.cls

```diff
@@ -150,7 +155,11 @@ public with sharing class LVL_LevelEdit_CTRL {
     */
     public PageReference saveAndNew() {
         try {
-            upsert lvl;
+            if (!reportFieldSetErrors()) {
```

Comment:
Might be a little more straightforward as:

```
if (reportFieldSetErrors()) {
    return null;
}
upsert lvl;
```

Reply:
I knew something felt "off" about that if statement 😅